src/qt/sendcoinsdialog.cpp

```diff
@@ -789,10 +789,8 @@ void SendCoinsDialog::coinControlChangeEdited(const QString& text)
         }
         else // Valid address
         {
-            CKeyID keyid;
-            addr.GetKeyID(keyid);
-            if (!model->havePrivKey(keyid)) // Unknown change address
-            {
+            const CTxDestination dest = addr.Get();
+            if (!model->IsSpendable(dest)) {
```

Comment:
Wouldn't `if (!model->IsSpendable(addr.Get())) {` be simpler?

Reply:
Yes it would, and that's how I had it until @MarcoFalke [made this comment](https://github.com/bitcoin/bitcoin/pull/11184#discussion_r135662398).

Is there a style guide that tells me how such things should be? Lots of people seem to have ideas about what's right, but I'd like to see something in writing, as it were.

Reply:
`dest` can be used in a few places below, but as I mentioned above this is just personal preference for one style over another. Just use what you feel makes more sense, no need to discuss.

Reply:
Good point. I use it in the few places below now too.